contrib/babelfishpg_tsql/antlr/TSqlParser.g4

```diff
@@ -3156,7 +3156,7 @@ column_def_table_constraint
 // empirically found: ROWGUIDCOL can be in various locations
 column_definition
     : simple_column_name (data_type system_versioning_column? | AS expression PERSISTED? ) ( special_column_option | collation | null_notnull )*
-      ( column_constraint? IDENTITY (LR_BRACKET sign? seed=DECIMAL COMMA sign? increment=DECIMAL RR_BRACKET)? )? for_replication? ROWGUIDCOL?
+      ( column_constraint? null_notnull? IDENTITY (LR_BRACKET sign? seed=DECIMAL COMMA sign? increment=DECIMAL RR_BRACKET)? )? for_replication? ROWGUIDCOL?
```

Comment:
How about just adding a '*'?:
```
column_constraint*  IDENTITY (.....
```
This will actually allow the SQL to have more constraints before IDENTITY keyword, something like below:
```
create table t1(a int primary key not null check(a % 2 = 0) IDENTITY(1,1), b int);
```
The above syntax is also supported in SQL server.
Let's try think of more such edge cases (if any) apart from what is already mentioned in the JIRA task.